modules/dr-restoring-cluster-state.adoc

```diff
@@ -314,3 +314,29 @@ etcd-ip-10-0-173-171.ec2.internal                2/2     Running     0
 ----
 
 Note that it might take several minutes after completing this procedure for all services to be restored. For example, authentication by using `oc login` might not immediately work until the OAuth server pods are restarted.
+
+[id="dr-scenario-cluster-state-issues_{context}"]
+= Issues and workarounds for restoring persistent storage state
+
+If your {product-title} uses persistent storage of any form, typically a state of the world is stored outside etcd. It could be an ElasticSearch cluster running in a pod or a database running in a statefulset. When restoring from an etcd backup, the status of the workloads in {product-title} is also restored. However, if the etcd snapshot is old, the status might be invalid or outdated.
```

Comment:
```suggestion
If your {product-title} cluster uses persistent storage of any form, a state of the world is typically stored outside etcd. It might be an Elasticsearch cluster running in a pod or a database running in a `StatefulSet` object. When restoring from an etcd backup, the status of the workloads in {product-title} is also restored. However, if the etcd snapshot is old, the status might be invalid or outdated.
```
^Note that you'll need to manually apply these changes and not click the "Accept change" button.

Also, not sure how familiar users are with the term "state of the world"?

Reply:
@gnufied consider Bob's comment about the "state of the world" term.

Reply:
we can reword it to say "state of the cluster" ?

Reply:
we missed this. @bobfuru @lpettyjo I thought we wanted to use word - "state of the cluster" ?